ruby/vampire_1st_attempt.rb

```diff
@@ -0,0 +1,79 @@
+puts "How many employees will be processed?"
+number_of_employees = gets.chomp.to_i
+
+until number_of_employees == 0 
+
+  puts "What is your name?"
+    suspect_name = gets.chomp
+  puts "How old are you?"
+    suspect_age = gets.chomp.to_i
+  puts "What year were you born?"
+    birth_year = gets.chomp.to_i
+  
+  case suspect_age
+  when suspect_age == (2017 - birth_year)
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.